corehq/apps/hqwebapp/views.py

```diff
@@ -67,92 +63,19 @@
 from corehq.apps.style.decorators import use_bootstrap3
 from corehq.apps.users.models import CouchUser
 from corehq.apps.users.util import format_username
+from corehq.form_processor.backends.sql.dbaccessors import FormAccessorSQL, CaseAccessorSQL
+from corehq.form_processor.exceptions import XFormNotFound, CaseNotFound
 from corehq.middleware import always_allow_browser_caching
 from corehq.util.datadog.const import DATADOG_UNKNOWN
 from corehq.util.datadog.metrics import JSERROR_COUNT
 from corehq.util.datadog.utils import create_datadog_event, log_counter, sanitize_url
 
 
-def pg_check():
-    """check django db"""
-    try:
-        a_user = User.objects.all()[:1].get()
-    except:
-        a_user = None
-    return (a_user is not None, None)
-
-
-def couch_check():
-    """Confirm CouchDB is up and running, by hitting an arbitrary view."""
-    try:
-        results = Application.view('app_manager/builds_by_date', limit=1).all()
-    except Exception:
-        return False, None
-    else:
-        return isinstance(results, list), None
-
-
 def is_deploy_in_progress():
     cache = get_redis_default_cache()
     return cache.get(DEPLOY_IN_PROGRESS_FLAG) is not None
 
 
-def celery_check():
-    try:
-        from celery import Celery
-        from django.conf import settings
-        app = Celery()
-        app.config_from_object(settings)
-        i = app.control.inspect()
-        ping = i.ping()
-        if not ping:
-            chk = (False, 'No running Celery workers were found.')
-        else:
-            chk = (True, None)
-    except IOError as e:
-        chk = (False, "Error connecting to the backend: " + str(e))
-    except ImportError as e:
-        chk = (False, str(e))
-
-    return chk
-
-
-def hb_check():
-    celery_monitoring = getattr(settings, 'CELERY_FLOWER_URL', None)
-    if celery_monitoring:
-        try:
-            cresource = Resource(celery_monitoring, timeout=3)
-            t = cresource.get("api/workers", params_dict={'status': True}).body_string()
-            all_workers = json.loads(t)
-            bad_workers = []
-            for hostname, status in all_workers.items():
-                if not status:
-                    bad_workers.append('* {} celery worker down'.format(hostname))
-            if bad_workers:
-                return (False, '\n'.join(bad_workers))
-            else:
-                hb = heartbeat.is_alive()
-        except Exception:
-            hb = False
-    else:
-        try:
-            hb = heartbeat.is_alive()
-        except Exception:
-            hb = False
-    return (hb, None)
-
-
-def redis_check():
-    try:
-        redis = cache.caches['redis']
-        result = redis.set('serverup_check_key', 'test')
-    except (InvalidCacheBackendError, ValueError):
-        result = True  # redis not in use, ignore
-    except:
-        result = False
-    return (result, None)
-
-
 def server_error(request, template_name='500.html'):
```

Comment:
Note that the messages here changed a bit, I'm going on the assumption that anything that uses this just checks the status code, or for the content to be `"success"`